src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -153,7 +153,7 @@ export default class FrameComponent extends Component<FrameComponentProps> {
         role="listitem"
         key={frame.id}
         className={className}
-        onMouseDown={e => this.onMouseDown(e, frame, selectedFrame)}
```

Comment:
i think this might actually be an issue

CC @bomsy 
Also, git blame / log could help

Reply:
I looked at a blame and the last I saw was conversion to JSX...

Reply:
It looks like the rationale was around focus states which is interesting
https://github.com/firefox-devtools/debugger/pull/1059

If we switch to onClick, we should update the handler as well to `this.onClick`